tests/unit/test_athena_prepared.py

```diff
@@ -0,0 +1,182 @@
+import logging
+
+import pytest
+from botocore.exceptions import ClientError
+
+import awswrangler as wr
+
+from .._utils import (
+    ensure_athena_query_metadata,
+    ensure_data_types,
+    get_df,
+    get_time_str_with_random_suffix,
+)
+
+logging.getLogger("awswrangler").setLevel(logging.DEBUG)
+
+pytestmark = pytest.mark.distributed
+
+
+@pytest.fixture(scope="function")
+def statement(workgroup0: str) -> str:
+    name = f"prepared_statement_{get_time_str_with_random_suffix()}"
+    yield name
+    try:
+        wr.athena.deallocate_prepared_statement(statement_name=name, workgroup=workgroup0)
+    except ClientError as e:
+        if e.response["Error"]["Code"] != "ResourceNotFoundException":
+            raise e
+
+
+def test_update_prepared_statement(workgroup0: str, statement: str) -> None:
+    wr.athena.prepare_statement(
+        sql="SELECT 1 AS col0",
+        statement_name=statement,
+        workgroup=workgroup0,
+    )
+
+    wr.athena.prepare_statement(
```

Comment:
Shouldn't these tests assert something based on the returned dict from the call? Otherwise how can we be sure that the update actually happened?

Reply:
The problem is that response doesn't give anything other that the response metadata:

```
{
  "ResponseMetadata": {
    "RequestId": "f00655d5-4512-4ee7-bdc3-c1af1fd5ba98",
    "HTTPStatusCode": 200,
    "HTTPHeaders": {
      "date": "Wed, 07 Jun 2023 14:04:02 GMT",
      "content-type": "application/x-amz-json-1.1",
      "content-length": "2",
      "connection": "keep-alive",
      "x-amzn-requestid": "f00655d5-4512-4ee7-bdc3-c1af1fd5ba98"
    },
    "RetryAttempts": 0
  }
}
```

So I think I might change the function to just return nothing. If the statement creation fails, the function will raise an error.